invenio_app_ils/records/serializers/__init__.py

```diff
@@ -9,10 +9,22 @@
 
 from invenio_records_rest.schemas import RecordSchemaJSONV1
 from invenio_records_rest.serializers.json import JSONSerializer
-from invenio_records_rest.serializers.response import record_responsify
+from invenio_records_rest.serializers.response import record_responsify, \
+    search_responsify
+
+from invenio_app_ils.records.serializers.item import ItemJSONSerializer
 
 json_v1 = JSONSerializer(RecordSchemaJSONV1, replace_refs=True)
 """JSON v1 serializer."""
 
+item_v1 = ItemJSONSerializer(RecordSchemaJSONV1, replace_refs=True)
```

Comment:
I think that we should not be loose using with [RecordSchemaJSONV1](https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/schemas/json.py#L39) probably strict and use [StrictKeysMixin](https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/schemas/json.py#L20).
This will ensure that we control exactly what fields we output.

WDYT?
@zzacharo WDYT?

Reply:
Yes that was my intention but back then everything was optional and undecided. You mean this no? https://github.com/inveniosoftware/invenio-app-ils/issues/136